### PR TITLE
Adding awesome-flights list

### PR DESCRIPTION
**Pull request by kevit**
**https://github.com/kevit/awesome-flights#readme**

**Explicit list of flight search engines, aeroport maps, low-fare aggregators etc.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖
Reviews
https://github.com/sindresorhus/awesome/pull/1456#issuecomment-448620638
https://github.com/sindresorhus/awesome/pull/1471#issuecomment-448620017

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by kevit** [comment]
fixed contribute.md

> Thanks for making an Awesome list! 
> 
> It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by sindresorhus** [comment]
> This pull request has a descriptive title in the format Add Name of List (Example: Add Swift), not Update readme.md or Add awesome list.

---

> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

---


The heading of your Awesome list should not be in slug format. `Awesome Unicorn`, not `awesome-unicorn`.

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/kevit/awesome-flights
✖ Linting

  README.md:9:3
  ✖   1:1  Missing License section                                                                    remark-lint:awesome/license
  ✖   1:1  Missing Awesome badge after the main heading                                               remark-lint:awesome/badge
  ✖   9:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  10:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  11:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  12:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  13:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  14:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  15:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  16:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  17:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  18:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  19:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  20:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  21:1  ToC item "Other services" does not match corresponding heading "Other services and lists"  remark-lint:awesome/toc
  ✖  21:3  Incorrect indentation before bullet: remove 3 spaces                                       remark-lint:list-item-bullet-indent
  ✖  40:3  Invalid list item link URL                                                                 remark-lint:awesome/list-item
  ✖  45:3  Invalid list item link URL                                                                 remark-lint:awesome/list-item

  18 errors
```

https://github.com/sindresorhus/awesome-lint

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
Some of the section titles are too long. Make them shorter and if they need more explanation, put it below the title as normal text.

**Comment by sindresorhus** [comment]
The description starts with an uppercase character and ends with a period. Same with other kinds of sentences.

**Comment by sindresorhus** [comment]
> This pull request has a descriptive title in the format Add Name of List (Example: Add Swift), not Update readme.md or Add awesome list.

**Comment by kevit** [comment]
almost all linting fixed, only internal links broken

```
✖ Linting

  README.md:39:3
  ✖  39:3  Invalid list item link URL  remark-lint:awesome/list-item
  ✖  44:3  Invalid list item link URL  remark-lint:awesome/list-item
```

**Comment by sindresorhus** [comment]
Bump

File: readme.md [hunk 1]
```diff
@@ -606,17 +606,18 @@
...
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Bitcoin Payment Processors](https://github.com/alexk111/awesome-bitcoin-payment-processors#readme) - Start accepting Bitcoin.
 
-
 ## Related
 
 - [Awesome Indexed](https://awesome-indexed.mathew-davies.co.uk) - Search the Awesome dataset.
 
 
+
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.
